app/src/main/java/com/realityexpander/tasky/core/presentation/notifications/RemindAtNotificationManager.kt

```diff
@@ -0,0 +1,332 @@
+            if (agendaItemType == AgendaItemType.Task.typeNameStr) {
+                NotificationCompat.Action
+                    .Builder(
+                        0,
+                        context.getString(R.string.agenda_notifications_complete_task_button),
+                        completeTaskPendingIntent
+                    ).build()
+            } else {
+                null
+            }
+
+        return completeTaskAction
+    }
+
+    private fun getInfoCardBitmap(
+        context: Context,
+        agendaItemTypeStr: String,
+        title: String,
+        description: String,
+        startDateTimeUtcMillis: Long,
+    ): Bitmap? {
+        val width = 400
+        val height = 300
+
+        // Create a Bitmap and a Canvas for the Bitmap
+        val bitmap = Bitmap.createBitmap(width, height, Bitmap.Config.ARGB_8888)
+        val canvas = Canvas(bitmap)
+        val paint = Paint()
+
+        // Draw the background
+        canvas.drawColor(Color.BLACK)
+
+        // Setup colors depending on Card Type
+        var backgroundColor = Color.WHITE
+        var textColor = Color.BLACK
+        when (agendaItemTypeStr.toAgendaItemType()) {
+            AgendaItemType.Event -> {
+                backgroundColor = context.resources.getColor(R.color.tasky_green, null)
+                textColor = Color.WHITE
+            }
+            AgendaItemType.Task -> {
+                backgroundColor = Color.LTGRAY
+                textColor = Color.BLACK
+            }
+            AgendaItemType.Reminder -> {
+                backgroundColor = context.resources.getColor(R.color.purple_200, null)
+                textColor = Color.WHITE
+            }
+            else -> context.resources.getColor(R.color.tasky_green, null)
+        }
+
+        val offsetYForTask = if (agendaItemTypeStr == AgendaItemType.Task.typeNameStr) 25 else 0
+
+        // Make rounded corners (Card)
+        paint.color = backgroundColor
+        paint.isAntiAlias = true
+        paint.style = Paint.Style.FILL
+        paint.strokeWidth = 10f
+        canvas.drawRoundRect(
+            0f, 50f + offsetYForTask,
+            width.toFloat(), height.toFloat() - (50f + offsetYForTask),
+            30f, 30f,
+            paint
+        )
+
+        // Set up for drawing text
+        paint.color = textColor
+        paint.textSize = 24f
+
+        val offsetY2ForTask = if (agendaItemTypeStr == AgendaItemType.Task.typeNameStr) 15 else 0
+
+        // Draw the description text on the Canvas
+        canvas.drawTextBlock(
+            "🔘 $title\n" +
+                    "• Starting at ${
+                        startDateTimeUtcMillis.toZonedDateTime()
+                            .format(DateTimeFormatter.ofPattern("h:mm a, E MMM d"))
+                    }" + "\n\n" +
+                    "• $description",
+            20f, 70f + offsetY2ForTask, width, height, paint
+        )
+        return bitmap
+    }
```

Comment:
Just curious, what does this draw? 😁

Reply:
This draws the card for the notification, like this:

[<img src="https://user-images.githubusercontent.com/5157474/206991659-fcd25421-6011-43db-a853-bfa4cbb3311d.png" width=250/>](https://user-images.githubusercontent.com/5157474/206991659-fcd25421-6011-43db-a853-bfa4cbb3311d.png)

Reply:
Ahh so you created a custom design for the Notification

Looks good, but im curious tho

What would happen if the title is too big or too small? Same with description
Ex; Description has 2k characters